installers/ansible/inventory

```diff
@@ -26,6 +30,14 @@ crunchy_debug='false'
 # ==================
 # kubernetes_context=''
 
+# Deploy with PGO-Installer
+# ==================
+# Note: Enable kubernetes_in_cluster if you are using
+# the pgo-installer image to install the operator
+# ==================
+# kubernetes_in_cluster=false
+# use_oc=false
```

Comment:
Remove this attribute, but perhaps replace with

`install_into_openshift`

or some other name.

Or perhaps this is a different flag, like

`pgo_installer_environment="openshift"`

and the default is to presume Kubernetes